docs/user-guide/framework/bootstrap-components/infrastructure.md

```diff
@@ -23,4 +27,131 @@ Ignition can bootstrap the following services for the Infrastructure API:
+        it is expected that the InfrastructureService will poll get_infrastructure_task on this driver to determine when the request has complete.
+
+        :param str infrastructure_id: identifier of the infrastructure to be removed
+        :param dict deployment_location: the location the infrastructure was deployed to
+        :return: an ignition.model.infrastructure.DeleteInfrastructureResponse
+        """
+        pass
+
+    @interface
+    def find_infrastructure(self, template, template_type, instance_name, deployment_location):
+        """
+        Finds infrastructure instances that meet the requirements set out in the given TOSCA template, returning the desired output values from those instances
+
+        :param str template: tosca template of infrastructure to be found
+        :param str template_type: type of template used i.e. TOSCA or Heat
+        :param str instance_name: name given as search criteria
+        :param dict deployment_location: the deployment location to deploy to
+        :return: an ignition.model.infrastructure.FindInfrastructureResponse
+        """
+        pass
+```
+
+There are several methods required on any Service implementing this Capability, in order to serve Create/Delete/Find infrastructure requests. The following sections describe the flow of an incoming request, outlines the role of Ignition and how your driver implementation is used.
+
+## On Create
+
+The InfrastructureApiService handles the incoming HTTP request, parsing the headers and body before making a call to the InfrastructureService with the expected arguments. 
+
+The InfrastructureService forwards this call down to the `create_infrastructure` method on the `InfrastructureDriver` provided by the user. 
+
+The `InfrastructureDriver` is expected to either:
+- Complete the request by initiating a creation of the infrastructure, returning immediately with an instance of `ignition.model.infrastructure.CreateInfrastructureResponse` which includes an `infrastructure_id` and `request_id` which may be used at a later time to check the status of the infrastructure being created. For example, in the Openstack VIM driver implementation, we return the Stack ID assigned to infrastructure.
+- Raise an Exception if the request is invalid or cannot be accepted. This Exception will end the continuation of this flow and instead result in an error being returned to the client of the InfrastructureApiService.
+
+On receival of a CreateInfrastructureResponse, the InfrastructureService will inform the InfrastructureTaskMonitoringService that it should monitor the completion of the infrastructure creation, using the `infrastructure_id` and `request_id` on the response.
+
+The InfrastructureTaskMonitoringService will periodically, using the JobQueueService (see [job queue](./job_queue)), check if infrastructure has been created. It will do this by polling the `get_infrastructure_task` method of the `InfrastructureDriver`
```

Comment:
Is it technically "polling" or just "calling" (polling may imply that it's sat there in a poll loop, perhaps)?